react/react1/week1/my-page/src/App.js

```diff
@@ -0,0 +1,20 @@
+import logo from "./logo.svg";
+import "./App.css";
+import { TodoList } from "./TodoList";
+
+const TodoData = [
+  { description: "Get out of bed", deadline: "Wed Sep 13 2017" },
+  { description: "Brush teeth", deadline: "Thu Sep 14 2017" },
+  { description: "Eat breakfast", deadline: "Fri Sep 15 2017" },
+];
+
+function App() {
+  return (
+    <div className="App">
+      <h3>Todo List</h3>
+      <TodoList TodoData={TodoData}></TodoList>;
```

Comment:
Usually we don't name the props as Pascal case, rather camel case. Pascal case is for components.